roles/postgres/files/postgres-backups@.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Back up the PostgresSQL database `%i`
+Requires=postgresql.service
+After=postgresql.service
+
+[Service]
+User = postgres
+Group = postgres
+WorkingDirectory=/var/lib/postgresql-backups
```

Comment:
nit: I wonder, should we move `/var/lib/postgresql-backups` to `postgres/defaults/main.yml`? So it can be parametrized (if needed)?

Reply:
Yeah, good point! Let me change this (it will require making this a Jinja template as well)